lib/zlib.js

```diff
@@ -574,6 +574,13 @@ Zlib.prototype._processChunk = function(chunk, flushFlag, cb) {
   req.callback = callback;
 
   function callback(availInAfter, availOutAfter) {
+    if (this) {
```

Comment:
I’d like it if there was a comment here saying that `this == req (== self._handle)`